src/utils/init/utils.ts

```diff
@@ -34,7 +34,7 @@ export const getPluginsToAutoInstall = (
   pluginsInstalled: string[] = [],
   pluginsRecommended: string[] = [],
 ) => {
-  const nextRuntime = '@netlify/plugin-nextjs'
+  const nextRuntime = '@opennextjs/netlify'
```

Comment:
This took me a while to understand — this is part of 'init' i.e. used in `ntl init` and related scenarios, and its purpose is to literally auto-install the plugin on the site via the Netlify API if it isn't already installed and we've detected that it should be. So it's OK to just update this name and not "handle" the old name in any way.

Reply:
... although, wait, isn't there a case like this?
1. next.js site exists today with @netlify/plugin-nextjs installed
2. we release this cli change
3. user clones their repo, runs `ntl init`, it checks and finds that `@opennextjs/netlify` is **not** auto-installed (even though `@netlify/plugin-nextjs` **is**) and auto-installs `@opennextjs/netlify`.
4. user has both plugins installed